src/patterns/start-using-a-service/index.md.njk

```diff
@@ -0,0 +1,75 @@
+description: Create a starting point for your digital service on GOV.UK
+section: Patterns
+theme: Help users to…
+aliases: start page, start pages
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+GOV.UK services must start on a GOV.UK content page. This links your service to the rest of GOV.UK, and means that users can find it using search or through GOV.UK’s navigation.
+
+## When to use this pattern
+
+Use this pattern to help you prototype a start point for your service, so you can test as much of the user journey as possible.
+
+The public-facing version of your service will need to start on a GOV.UK content page, published using one of the GOV.UK publishing apps. You’ll usually need to agree content for this GOV.UK content page with the GDS content team. Read [guidance on how to get your service on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk).
+
+## How it works
+
+A service's start point should:
+- give the user just enough information to help them understand what the service does and whether it will meet their need — including [giving the service a name that reflects the problem it solves for users](https://www.gov.uk/service-manual/design/naming-your-service)
+- include a ‘button’ linking into the service, with text that’s consistent with the action you’re asking users to take — for example, ‘Start now’, ‘Sign in’ or ‘Register or update your details’ (if you need a secondary call to action, use a standard link)
+- let users sign in, resume an application they’ve already started or update their details (if relevant)
+- include any other information that most users are likely to need before they start using the service online — for example, how much it costs to use the service and roughly how long it will take
+- include details of other ways to access the service — for example, by telephone or by completing a paper form
+
+You’ll also need to list any documents or information the user will need to help them complete the service (there’s no need to list information that a user is likely to know from memory — for example, their own date of birth).
+
+You can list the documents or information at the start point, if it does not make the page too complicated. Or you can move these details to a separate ‘What you’ll need’ page, served within the service subdomain.
+
+## Eligibility
+
+Avoid using the start point to provide complex eligibility information. Instead, ask questions inside the service to find out whether the user is eligible.
+
+Read [guidance on designing how GOV.UK content and transactions work together](https://www.gov.uk/service-manual/design/govuk-content-transactions).
+
+## Choosing a format for a start point on GOV.UK
+
+The options available depend on whether the start point will be published as ‘mainstream’ or ‘Whitehall’ content.
+
+‘Mainstream’ content is content aimed at the general public — or at meeting business needs that are not sector-specific.
```

Comment:
This is the second use of mainstream, does it need to be in quotes?

Reply:
Good catch!

```suggestion
Mainstream content is content aimed at the general public — or at meeting business needs that are not sector-specific.
```